submissions/MosQuitO404/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hooli-style-Popup</title>
+</head>
+
+<body>
+    <header class="popup-header">
+        <nav class="nav-items">
+            <ul class="nav-items-ul" id="nav-items-ul-first">
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Home</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">About</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Pricing</a>
+                </li>
+            </ul>
```

Comment:
All these elements must be a part of the one `ul` element.
![image](https://user-images.githubusercontent.com/16196199/185801825-1d4af3c5-ed9f-44d4-af83-ad63c595a003.png)

Reply:
Hi! Bug were fixed. Review pls.